projects/aca-playwright-shared/src/api/tags-api.ts

```diff
@@ -72,4 +74,23 @@ export class TagsApi {
       return null;
     }
   }
+
+  async listTags(params?: { tag?: string; matching?: boolean }): Promise<TagPaging> {
```

Comment:
I think there is no sense to create separate function just to wrap listTags function from TagsApi. I can see it's used now only in single place (below in deleteTagsByTagName function). Actually that extra wrapper doesn't give any value because in deleteTagsByTagName you have already try catch so you can call listTags from TagsApi in deleteTagsByTagName function directly and thanks that here will be less redundant code.